services/user.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None,
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(pk=user_id)
+
+
+def update_user(
+    user_id: int,
+    username: Optional[str] = None,
+    password: Optional[str] = None,
+    email: Optional[str] = None,
+    first_name: Optional[str] = None,
+    last_name: Optional[str] = None,
+) -> None:
+    user = User.objects.get(id=user_id)
```

Comment:
Using `get()` without handling exceptions can lead to `DoesNotExist` errors if the `User` is not found. Consider using `get_object_or_404()` or handling the exception to provide a more user-friendly error message.